pkgs/data/fonts/lucide/default.nix

```
@@ -0,0 +1,33 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchurl,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "lucide";
+  version = "0.541.0";
+
+  src = fetchurl {
+    url = "https://unpkg.com/lucide-static@${version}/font/Lucide.ttf";
+    sha256 = "sha256-H8ACnTkbR4N3yYbWuteZoqfPlLu+OGm/NEfweBad8uM=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/fonts/truetype
+    cp -a *.ttf $out/share/fonts/truetype/
+  '';
+
+  sourceRoot = ".";
+  unpackCmd = ''
+    ttfName=$(basename $(stripHash $curSrc))
+    cp $curSrc ./$ttfName
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
Nit: Avoid [wide `with` scopes](https://nix.dev/guides/best-practices#with-scopes) in attribute sets. It's alright to repeat `lib.*` when you need it:
```nix
{
  meta = {
    license = lib.licenses.publicDomain;
    maintainers = with lib.maintainers; [ maintainer ];
  };
}
```